backend/test/test-run.sh

```
@@ -0,0 +1,63 @@
+
+# Get API URL
+echo "Fetching route to $DEPLOYMENT_NAME"
+timeout 1m bash -c "until oc -n $NAMESPACE get route "$DEPLOYMENT_NAME" &> /dev/null; do echo 'Waiting for the route to $DEPLOYMENT_NAME...'; sleep 10; done"
+export API_URL="https://$(oc -n $NAMESPACE get route "$DEPLOYMENT_NAME" -o jsonpath={.spec.host})"
+
+# Get API Token
+echo "Generate Token for $DEPLOYMENT_NAME"
+export API_TOKEN=$(oc create token "$DEPLOYMENT_NAME" --namespace "$NAMESPACE" --duration=60m)
+
+# Run Tests
+cd $TEST_DIRECTORY
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)